requirements/base.txt

```diff
@@ -20,6 +20,12 @@ pyopenssl==22.0.0
 
 # Needed for supporting Protocol in Python 3.7, Protocol class became public with python3.8
 typing_extensions==3.10.0.0
+# Typeshed type stubs
+types-pywin32==304.0.0.1
```

Comment:
will this cause issues when installing on a linux/mac system, the classifiers on the project indicate it to be just for windows.

Reply:
@Avasam Can you please address this feedback? Thanks!

Reply:
Thanks for the ping, I forgot about this.

The stub package only contains type stubs (`.pyi files`). There's no functional code.
At worst they'll keep the typing errors in their IDE with `pywintypes` if they *don't* install it.